### ISSUE TITLE
Error thrown when plotting line with ordinal x values

### ISSUE THREAD

**Issue by ChakriCherukuri** [bloomberg/bqplot#770]
```python
import bqplot.pyplot as plt

f = plt.figure()
plt.plot(list('ABC'), [1, 2, 3], 'ms-')
f
```
Running the above code throws the following error:
TraitError: Array type not supported for trait x of class <class 'bqplot.marks.Lines'>: expected a             array of kind in list ['b', 'i', 'u', 'f', 'M'] and got an array of type <U1 (kind U)

This has to do with 'array_supported_kinds()' which was added to x and y attributes of various marks. @maartenbreddels can you please check?

**Comment by maartenbreddels** [comment]
Hi Chakri,

Indeed a bug, I'm working on a PR to test/fix this.

cheers,

Maarten

**Comment by SylvainCorlay** [comment]
Fixed by #772.
